From ed37db420aec83c6530c040cfc79e515b36cffa9 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Tue, 20 Aug 2024 14:20:20 +0200
Subject: [PATCH 1059/1695] dts: stm32: update ETHCK on STM32MP215F-DK board

Set ETHCK frequency from 125MHz to 50MHz

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 19650d1022abcce98203f1f48b7ee080c674adc5 dts: stm32: update ETHCK on STM32MP215F-DK board]
Signed-off-by: Stephane Danieau <stephane.danieau@st.com>
Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Change-Id: I9df0ae9be19d63199e77a9baac18558ab795cdd8
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/399120
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rcc.dtsi
index 6b36a8e0a..c10032999 100644
--- a/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rcc.dtsi
@@ -88,7 +88,7 @@
 		FLEXGEN_CFG(51, XBAR_SRC_PLL4, 0, 5)
 		FLEXGEN_CFG(52, XBAR_SRC_PLL4, 0, 5)
 		FLEXGEN_CFG(53, XBAR_SRC_PLL4, 0, 5)
-		FLEXGEN_CFG(54, XBAR_SRC_PLL6, 0, 3)
+		FLEXGEN_CFG(54, XBAR_SRC_PLL6, 0, 9)
 		FLEXGEN_CFG(55, XBAR_SRC_PLL6, 0, 3)
 		FLEXGEN_CFG(56, XBAR_SRC_PLL4, 0, 5)
 		FLEXGEN_CFG(57, XBAR_SRC_HSE_KER, 0, 1)
-- 
2.47.2

